Formula/tox.rb

```diff
@@ -31,6 +31,11 @@ class Tox < Formula
     sha256 "18d82244ee114f543149c66a6e0c14e9c4f8a1044b5cdaadd0f82159d6a6ff59"
   end
 
+  resource "importlib-metadata" do
+    url "https://files.pythonhosted.org/packages/e2/ae/0b037584024c1557e537d25482c306cf6327b5a09b6c4b893579292c1c38/importlib_metadata-1.7.0.tar.gz"
+    sha256 "90bb658cdbbf6d1735b6341ce708fc7024a3e14e99ffdc5783edea9f9b077f83"
+  end
```

Comment:
Are you sure you're running Python 3.8 when you saw this dependency? It's supposed to only be there for < 3.8: https://github.com/tox-dev/tox/blob/db8cfba41ad34493e7d5de4ac1a55d3c264c0a13/setup.cfg#L47. I think `zipp` has a similar story.

Reply:
Hmm...

Reply:
Ah, you're right I needed to run `brew link`